stonesoup/non_state_measures.py

```diff
@@ -0,0 +1,169 @@
+import heapq
```

Comment:
Looking at the docs, a lot of these are quite messy when rendered. Would it be possibly to make some changes to make them look a bit cleaner/add missing descriptions for parameters?

Quite a few full stops need adding in as well.